src/configs/rate-limiting-config.ts

```diff
@@ -0,0 +1,10 @@
+import memjs from "memjs";
+
+const memcachedUrl = `${process.env.MEMCACHED_CLIENT_USERNAME}:${process.env.MEMCACHED_CLIENT_PASSWORD}@${process.env.MEMCACHED_CLIENT_SERVERS}`;
+const memcachedClient = memjs.Client.create(memcachedUrl);
+
+export default {
+  windowMs: 300000,
+  requestsUntilBlock: 200,
+  memcachedClient,
+};
```

Comment:
I extracted the config needed by the rate limiting middleware...

Reply:
Good idea :).